### PR TITLE
jr-MS-1463 add CYA page logic

### PR DESCRIPTION
**Pull request by jjrobothamho**
- update behaviour name
- add logic for CYA page

### CODE REVIEW

**Comment by jjrobothamho** [comment]
dependent on #390

File: apps/nrm/behaviours/get-saved-reports.js [hunk 1]
```diff
@@ -37,8 +37,9 @@ const getReports = async(req) => {
 
     // Check the user has an report
     if (savedData) {
+
       // set the session data with previously saved data
-      setSessionData(req, savedData);
+      setSessionData(req, JSON.parse(savedData.json_saved_data));
```
Comment (sulthan-ahmed) [inline apps/nrm/behaviours/get-saved-reports.js]: erm you are doing JSON.parse on json saved data? 🤔

File: apps/nrm/behaviours/next-steps/page-custom-next-steps.js [hunk 2]
```diff
@@ -227,6 +227,22 @@ module.exports = {
...
+  checkYourAnswersSoFar: (req) => {
+    let nextStep = '/nrm/start';
+
+    const savedData = req.sessionModel.get('report-read-success');
+
+    if (savedData) {
+      // get the last saved page - only caveat is that the user would have already answered this question
+      // ideally we would like to send them to the subsequent page
+      const visitedPages = savedData.visited_pages.split(',');
+      const getLastSavedPage = visitedPages[visitedPages.length - 1].trim();
```
Comment (sulthan-ahmed) [inline apps/nrm/behaviours/next-steps/page-custom-next-steps.js]: when they click save, it should really save the form data but we can do that later

File: apps/nrm/behaviours/get-saved-reports.js (hunk 1 above)
Comment (jjrobothamho) [reply apps/nrm/behaviours/get-saved-reports.js]: Yeah, it gets stored as a string.

Comment (sulthan-ahmed) [reply apps/nrm/behaviours/get-saved-reports.js]: ok I guess it's fine.  It just confused me as it's called json so I thought you were passing json but what you mean is your passing a json string

**Pull request merged**